src/Tempest/Database/DatabaseDriver.php

```diff
@@ -11,4 +11,8 @@ public function getDsn(): string;
     public function getUsername(): ?string;
 
     public function getPassword(): ?string;
+
+    public function createQueryStatement(string $table): QueryStatement;
```

Comment:
Perhaps we can even move this to the `DatabaseDialect` enum, and simply have an `getDialect` on the driver class?

Reply:
The `createQueryStatement(..)` needs a instance of the database driver. I'm not sure how to pass that through the enum.